frontend/src/components/Mesh/TestTracingConfig.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { useKialiTranslation } from '../../utils/I18nUtils';
+import { Button, ButtonVariant, Spinner, Tooltip, TooltipPosition } from '@patternfly/react-core';
+import AceEditor from 'react-ace';
+import { Theme } from '../../types/Common';
+import { istioAceEditorStyle } from '../../styles/AceEditorStyle';
+import { aceOptions, yamlDumpOptions } from '../../types/IstioConfigDetails';
+import { getKialiTheme } from '../../utils/ThemeUtils';
+import { dump, loadAll } from 'js-yaml';
+import ReactAce from 'react-ace/lib/ace';
+import { ValidationTypes } from 'types/IstioObjects';
+import { kialiStyle } from '../../styles/StyleUtils';
+import { Validation } from '../Validations/Validation';
+import * as API from '../../services/Api';
+
+type CheckModalProps = {
+  configData?: unknown;
+};
+
+const healthStatusStyle = kialiStyle({
+  marginLeft: '0.5rem'
+});
+
+export const TestTracingConfig: React.FC<CheckModalProps> = (props: CheckModalProps) => {
+  const { t } = useKialiTranslation();
+  const [loading, setLoading] = React.useState(false);
+  const [error, setError] = React.useState<string | null>(null);
+  const [isModified, setIsModified] = React.useState(false);
+  const aceEditorRef = React.useRef<ReactAce | null>(null);
+  const [source, setSource] = React.useState<string>(dump(props.configData, yamlDumpOptions));
+
+  const theme = getKialiTheme();
+  const [configResult, setConfigResult] = React.useState<string | null>(null);
+
+  const parseYamlDocumentsSync = (yamlText: string): any[] => {
+    let document: any;
+    loadAll(yamlText, doc => {
+      document = doc;
+    });
+    return document;
+  };
+
+  const testConfig = (): void => {
+    const objectModified = parseYamlDocumentsSync(source);
```

Comment:
And here check if YAML was loaded correctly
if (!objectModified) {
      return
    }

Reply:
Also included